Exercise_A/index.html

```diff
@@ -10,6 +10,7 @@
   </head>
   <body>
     <!-- CODE HERE -->
+    <a href="about.html"> <img src="../image/flower-icon.png" alt="flower"></a>
 
     <script src="" async defer></script>
   </body>
```

Comment:
Hello Asis 
Great work with this exercise, the link works when the image is clicked